lesson05/main.js

```diff
@@ -1,17 +1,24 @@
-const ul = document.getElementById("js-ul"); 
-const fragment = document.createDocumentFragment(); 
 const elements = [ 
   {to: "bookmark.html", img:"img/1.png", alt:"画像1", text: "ブックマーク"}, 
   {to: "message.html", img:"img/2.png", alt:"画像2", text: "メッセージ"} 
 ]; 
-elements.forEach(function(element){ 
-  const li = document.createElement("li"); 
-  const a = document.createElement("a"); 
-  const img = document.createElement("img"); 
-  a.href = element.to; 
-  a.textContent = element.text; 
-  img.src = element.img; 
-  img.alt = element.alt; 
-  fragment.appendChild(li).appendChild(a).insertAdjacentElement("afterbegin",img); 
-}); 
-ul.appendChild(fragment);
+
+let promise =  new Promise((ok)=> {
+  ok(elements);
+});
+
+promise.then((elements)=> {
+  const ul = document.getElementById("js-ul"); 
+  const fragment = document.createDocumentFragment(); 
+  elements.forEach(function(element){ 

```

Comment:
[ IMO ]
Since you use the Arrow function elsewhere, I thought it would be a good idea to align it as well.
```suggestion
  elements.forEach(element => { 
```

Reply:
Thank you haruna-san :)
I apliciate your many advices :)